src/interpreter_funcs/voxelize.rs

```diff
@@ -84,7 +86,7 @@ impl Func for FuncVoxelize {
                 heavier geometry that significantly affect performance. Too high values produce \
                 single large voxel, too low values may generate holes in the resulting geometry.",
                 refinement: ParamRefinement::Float3(Float3ParamRefinement {
-                    min_value: Some(f32::MIN_POSITIVE),
+                    min_value: Some(0.005),
```

Comment:
Small values, even though they may not generate too many voxels, cause the calculation to be too small. It's because the `to_mesh` function needs to divide each face into chunks as small as the smallest voxel dimension, which makes the voxelization too slow. `0.005` is an arbitrary number but seems to work well. The minimum value is being set very often because the user drags the slider all the was to the left quite often.